docs/security/trust/content_trust.md

```diff
@@ -221,16 +221,16 @@ Once you have a root key on your system, subsequent images repositories
 you create can use that same root key:
 
 ```bash
-$ docker push docker.io/docker/seaside:latest
-The push refers to a repository [docker.io/docker/seaside] (len: 1)
+$ docker push docker.io/<username>/<someimage>:latest
```

Comment:
Ah sorry, yes good point, `nottrusttest` implies it's not trusted.  I was going for "any image not named trusttest", but missed the double meaning.  Maybe just `otherimage` without the brackets?